app/views/items/index.html.erb

```diff
@@ -45,6 +45,11 @@
           <tr class="item">
             <td class="itemname">
               <%= link_to "#{p.name}", item_path(p) %>
+               <% if user_signed_in? %>
+                <% if current_user.favorites.include? p %>
+                  <%= image_tag("favorite.jpg",size: '10')%> 
```

Comment:
can you show how this looks like in your PR description?

Reply:
+1